libs/fluxc-plugin/src/test/java/org/wordpress/android/fluxc/store/WCStatsStoreTest.kt

```diff
@@ -385,7 +376,13 @@ class WCStatsStoreTest {
                     any(),
                     any()
                 )
-            ).thenReturn(FetchRevenueStatsResponsePayload(it, StatsGranularity.DAYS, WCRevenueStatsModel()))
+            ).thenReturn(
+                FetchRevenueStatsResponsePayload(
+                    it,
+                    StatsGranularity.DAYS,
+                    WCRevenueStatsModel(LocalId(1), "", "", "", "", "", "")
```

Comment:
nit: I've noticed `WCRevenueStatsModel(LocalId(1), "", "", "", "", "", "")` is repeated in this PR. WDYT about creating a factory, **test-only** extension method? I'm thinking about

```kotlin
fun WCRevenueStatsModel.Companion.fake() = WCRevenueStatsModel(LocalId(1), "", "", "", "", "", "")
```

and then 

```suggestion
                    WCRevenueStatsModel.fake()
```

Or you can even make it more customisable like `revenue` method from `RevenueStatsDaoTest`

Reply:
Thanks for the suggestion @wzieba , will do, I also like how you've named it `fake()`! 💯

Reply:
FYI: This is now done: 92d5229bf68060306e9fd5b6da6d0add2ea5123a Cc @wzieba